salesforce/management/commands/sync_reviews.py

```diff
@@ -18,7 +18,7 @@ def handle(self, *args, **options):
             for record in sf_reviews:
                 try:
                     review = PartnerReview.objects.get(review_salesforce_id=record['Id'])
-                    if review.status != 'Edited':
+                    if review.status != 'Edited' and review.status != 'Deleted':
```

Comment:
I think you want an `or` here

Reply:
if I use `or` and the status is `Edited`, the left side of the `or` is false so it goes to the right side which is true and an `Edited` status gets processed which we do not want at that point. Using `and`, both sides have to be true for the review to be processed.

Reply:
gotcha, you are correct!